### PR TITLE
[select2] Automatically include i18n resource file

### PR DESCRIPTION
**Pull request by tgoetz**
... based on configured (or detected) language

### CODE REVIEW

File: select2-parent/select2/src/main/java/org/wicketstuff/select2/Select2ResourcesBehavior.java [hunk 1]
```diff
@@ -60,20 +56,5 @@ public void renderHead(Component component, IHeaderResponse response)
 		{
 			response.render(CssHeaderItem.forReference(settings.getCssReference()));
 		}
-		//i18n
-		try
-		{
-			String name = String.format("res/js/i18n/%s.js", Session.get().getLocale().toLanguageTag());
-			URL lang = getClass().getResource(name);
-			if (lang != null)
```
Comment (solomax) [inline select2-parent/select2/src/main/java/org/wicketstuff/select2/Select2ResourcesBehavior.java]: Previously there was a check: if i18n file for given locale exists, currently such check is missing ....

Comment (tgoetz) [inline select2-parent/select2/src/main/java/org/wicketstuff/select2/Select2ResourcesBehavior.java]: Yes, but do we really need such a ckeck?

1. It adds overhead to the (guessed) more than ~90% where the configured language is correct (and existing)
2. There is a fallback to "en" anyway
3. You'll see a 404 request if you configured a non-existing language
4. User with a browser that uses an unsopported language will *always* see a warning in the logfile ...

Comment (tgoetz) [inline select2-parent/select2/src/main/java/org/wicketstuff/select2/Select2ResourcesBehavior.java]: Ok, thinking again I got to the point that I might be a good idea to have such a check, so I added one (see latest commit).

File: select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java [hunk 2]
```diff
@@ -42,10 +44,13 @@
...
 	}
 
+	// language code (e.g. "en", "de", "fr", ...)
+	private String language;
+
 	private Integer minimumInputLength, minimumResultsForSearch;
 	private Integer maximumSelectionLength;
-	private Object placeholder;
-	private boolean allowClear;
+	private Object placeholder; // ok
```
Comment (martin-g) [inline select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java]: What is `// ok` ?

Comment (tgoetz) [inline select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java]: Oops, shouldn't have made it into the commit, was about to check which settings/options are still valid with select2 4.x. Will remove this in a minute ...

Comment (solomax) [inline select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java]: Great :)
I'm ready to merge :)

**Comment by solomax** [comment]
Thanks :)

**Pull request merged**